### PR TITLE
Default financial fields to 0 and make them optional

### PR DESCRIPTION
**Pull request by jbalboni**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3061

This was mostly done in the backend schema, so not many changes here. There are some tweaks/clean up of the additional sources and financials pages, too.

screen shot 2017-06-01 at 1 53 43 pm

### CODE REVIEW

File: src/js/pensions/components/DisclosureTitle.jsx [hunk 1]
```diff
@@ -3,11 +3,10 @@ import _ from 'lodash/fp';
 
 export default function createDisclosureTitle(path, title) {
   return function DisclosureTitle({ id, formData }) {
-    const nameData = _.get(path, formData);
-    const name = nameData ? `${nameData.first} ${nameData.last}` : null;
+    const { first, last } = _.get(path, formData) || {};
     return (
       <div>
-        <h4 className="pensions-disclosure-name">{name}</h4>
+        <h4 className="pensions-disclosure-name">{first} {last}</h4>
```
Comment (jbalboni) [inline src/js/pensions/components/DisclosureTitle.jsx]: Was getting `undefined undefined` before, this fixes that.

**Review by cvalarida — APPROVED** [review]
I chuckled a little when I read "not too many changes here" and saw 43 commits. 😄

Looks good!

**Comment by jbalboni** [comment]
@cvalarida I blame git for that.

**Comment by cvalarida** [comment]
Git's a git :man_shrugging:

**Pull request merged**